components/markets/MarketAssetDetails.tsx

```diff
@@ -159,11 +158,10 @@ const MarketAssetDetails = observer(({ marketId }: { marketId: number }) => {
   };
 
   const getDisputedCategoricalOutcome = () => {
+    if (!rpcMarket) return;
```

Comment:
Is this needed? Doesn't look like the function uses it

Reply:
Removed. I was doing some testing as I noticed similar functions had it.